tests/dialects/test_mpi.py

```diff
@@ -4,15 +4,16 @@
 
 
 def test_mpi_baseop():
+    """
+    This test is used to track changes in `.get` and other accessors
+    """
     alloc0 = memref.Alloc.get(f64, 32, [100, 14, 14])
     dest = Constant.from_int_and_width(1, i32)
-    send = mpi.ISend.get(alloc0, dest, 1)
-    recv = mpi.IRecv.get(dest, alloc0.memref, 1)
+    unwrap = mpi.UnwrapMemrefOp.get(alloc0)
+    tag = Constant.from_int_and_width(1, i32)
+    send = mpi.ISend.get(unwrap.ptr, unwrap.len, unwrap.typ, dest, tag)
+    wait = mpi.Wait.get(send.request, ignore_status=False)
+    recv = mpi.IRecv.get(unwrap.ptr, unwrap.len, unwrap.typ, dest, tag)
     test_res = mpi.Test.get(recv.request)
-    code2 = mpi.Wait.get(recv.request)
-
-    assert send.operands[0] is alloc0.results[0]
-    assert send.operands[1] is dest.results[0]
-    assert recv.operands[0] is send.operands[1]
-    assert code2.operands[0] is recv.results[0]
-    assert test_res.operands[0] is recv.results[0]
+    source = mpi.GetStatusField.get(wait.status,
```

Comment:
Do we not want the asserts?

Reply:
I think this test is really just to have a check that the `get`s work. I can def see an argument for having the asserts in. On the other hand, I haven't found them to be useful for myself while refactoring.

Reply:
I had a thought about it and added the asserts back, I think they are doing a somewhat important part in testing.